distributed/comm/ucx.py

```diff
@@ -107,6 +109,37 @@ def device_array(n):
             pool_allocator=True, managed_memory=False, initial_pool_size=pool_size
         )
 
+def get_create_endpoint():
+    global reuse_endpoints_create
+    if reuse_endpoints_create is None:
+        init_once()
```

Comment:
Would it make sense to move this function into `init_once` as opposed to calling it from here? May simplify some of the logic as well.

Reply:
Right, it simplifies the code quite a bit :)